### PR TITLE
feat: adds logs rollover support

### PR DESCRIPTION
**Pull request by pedrolamas**
Adds support for application logs rollover

## Button with tooltip

image

## Available options

image

## Success message

image

## Failure message (text provided by Moonraker)

image

### CODE REVIEW

File: src/components/common/RolloverLogsDialog.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<template>
+  <app-dialog
+    v-model="open"
+    :title="$t('app.general.title.rollover_logs')"
+    :save-button-text="$t('app.general.btn.accept')"
+    :max-width="450"
+    @save="sendAccept"
+  >
+    <v-card-text>
+      <v-radio-group v-model="application">
```
Comment (matmen) [inline src/components/common/RolloverLogsDialog.vue:10]: image
```suggestion
      <v-radio-group
        v-model="application"
        class="mt-0"
        hide-details
      >
```

**Review by matmen — APPROVED** [review]
LGTM functionality wise, I just think the extra space around the radio group looks a bit odd:

File: src/components/widgets/system/SystemOverviewCard.vue [hunk 2]
```diff
@@ -3,6 +3,27 @@
     :title="$t('app.general.title.system_overview')"
     icon="$desktopTower"
   >
+    <template #menu>
+      <v-tooltip
+        bottom
+      >
+        <template #activator="{ on, attrs }">
+          <app-btn
```
Comment (matmen) [inline src/components/widgets/system/SystemOverviewCard.vue:11]: Eh, I just realized that rolling over the logs restarts the service(s) - so this button should probably be disabled when the printer is busy..

**Review by matmen — CHANGES_REQUESTED** [review]

File: src/components/widgets/system/SystemOverviewCard.vue (hunk 2 above)
Comment (pedrolamas) [reply src/components/widgets/system/SystemOverviewCard.vue:11]: Oh, very good point!!

Comment (pedrolamas) [reply src/components/widgets/system/SystemOverviewCard.vue:9]: Done!

File: src/components/common/RolloverLogsDialog.vue (hunk 1 above)
Comment (pedrolamas) [reply src/components/common/RolloverLogsDialog.vue]: I've also made the dialog width a bit smaller (450 -> 400) and updated the main screenshot on this PR.

File: src/components/widgets/system/SystemOverviewCard.vue [hunk 3]
```diff
@@ -3,6 +3,26 @@
...
+      <v-tooltip bottom>
+        <template #activator="{ on, attrs }">
+          <app-btn
+            v-bind="attrs"
+            color=""
+            fab
+            x-small
+            text
+            class="ms-1 my-1"
+            :disabled="printerPrinting || printerPaused"
```
Comment (matmen) [inline src/components/widgets/system/SystemOverviewCard.vue:16]:
```suggestion
            :disabled="printerBusy"
```

**Review by matmen — COMMENTED** [review]
I'm not sure, but I think `printerPrinting || printerPaused` can still kill the printer when homing or calibrating, no? Won't `printerBusy` be better suited here?

File: src/components/widgets/system/SystemOverviewCard.vue (hunk 3 above)
Comment (pedrolamas) [reply src/components/widgets/system/SystemOverviewCard.vue]: Good point, changed!

**Review by matmen — APPROVED** [review]

**Pull request merged**